luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -3,57 +3,24 @@
 function gadget:GetInfo()
 	return {
 		name = "Team Power Watcher",
-		desc = "Tracks power of individual and total units per team. To be used for PvE dynamic difficulty and library functions to assertain aspects of game progression",
+        desc = "Tracks power of individual and total units per team. To be used for PvE dynamic difficulty and library functions to assertain aspects of game progression",
 		author = "SethDGamre",
 		date = "2024-07-12",
-		license = "None",
 		layer = 1,
 		enabled = true
 	}
 end
 
-if gadgetHandler:IsSyncedCode() then
+if not gadgetHandler:IsSyncedCode() then return end
 
-	local spGetGameFrame          = Spring.GetGameFrame
-	local spSetUnitRulesParam     = Spring.SetUnitRulesParam
-	local spGetUnitRulesParam = Spring.GetUnitRulesParam
-	local spGetTeamResources = Spring.GetTeamResources --(teamID, "metal"|"energy") return nil | currentLevel
-	local spGetUnitDefID        = Spring.GetUnitDefID
 
-	local spGetTeamList			= Spring.GetTeamList
-	local spGetUnitTeam 		= Spring.GetUnitTeam
+--use VFS.Include("luarules/gadgets/game_team_power_watcher.lua") assuming filepath is still correct to gain access to the data within this gadget and use its functions
+GG.PowerLib = {
 
-	local spGetGameSeconds = Spring.GetGameSeconds
+    --global variables
 
-	local random = math.random
-	local math_min = math.min
-	local sin    = math.sin
-	local cos    = math.cos
-
-	local GAME_SPEED = Game.gameSpeed
-
-	local unitsWithPower = {}
-	local teamList = spGetTeamList()
-	local neutralTeamNumber
-	local teamPowers = {}
-	local highestTeamPower = {}
-    local averageTeamPower = 0
-    local averageAlliedTeamPower = 0
-    local averageHumanTeamPower = 0
-    local averageTechGuesstimate = 0
-    local averageAlliedTechGuesstimate = 0
-    local averageHumanTechGuesstimate = 0
-    local peakTeamPowers = {}
-
-    --AI team lists
-    local scavTeam
-    local raptorTeam
-    local AIteams = {}
-    local humanTeams = {}
-
-
-    --used for tech level guesstimate functions
-    local powerThresholds = {
+--used for tech level guesstimate functions last updated 7/12/24
+    PowerThresholds = {
```

Comment:
Although cosmetic only, I would switch the order of the two properties so that techLevel comes first, since it appears techLevel is the independent variable and threshold is the dependent variable.